changelogs/fragments/82383-fix-boot-time-command-on-freebsd.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - reboot - change the boot time command on FreeBSD because the current one is not stable (https://github.com/ansible/ansible/pull/82383)
```

Comment:
Note that this appears in the changelog, along with other entries. Release notes generally refer to things that changed compared to the previous release. In that context, the end-users (the main target audience reading the changelog) wouldn't understand what's "the current one" in that context.

Could you try re-phrasing it?